test/unit/spec/loadingWidgetSpec.js

```diff
@@ -11,10 +11,6 @@ define([
         '/narrative/static/kbase/images/kbase_animated_logo.gif'
     );
     describe('Test the LoadingWidget module', () => {
-        beforeAll(() => {
-            document.body.innerHTML = '';
-        });
-
```

Comment:
remove this as the loadingWidget staying in a single node, so there's no need to mess with the document.